src/librustc_resolve/lib.rs

```diff
@@ -3299,9 +3299,11 @@ impl<'a, 'crateloader: 'a> Resolver<'a, 'crateloader> {
                         err.span_label(base_span,
                                        "expecting a type here because of type ascription");
                         if line_sp != line_base_sp {
-                            err.span_suggestion_short(sp,
+                            err.span_suggestion_short_with_applicability(sp,
                                                       "did you mean to use `;` here instead?",
-                                                      ";".to_string());
+                                                      ";".to_string(),
+                                                      Applicability::Unspecified,
+                                                      );
```

Comment:
Fix indentation

Reply:
Stepping back further than 4 spaces because of 100-chars line limit.